content/en/docs/guides/runbooks/orca-zombie-executions.md

```diff
@@ -104,7 +104,13 @@ If any part of an Execution cannot be re-hydrated, the entire Execution will be
 ## Cancel the Execution
 
 If the Execution cannot be rehydrated, it will need to be canceled.
-You can cancel the Execution via the UI or force cancellation via an Orca admin API:
+
+> Note: Prior to Spinnaker 1.32.0, cancelling a zombied execution via the UI
```

Comment:
We're in a strange in-between state...where 1.31.0 hasn't been released yet, but the branches are cut and it may have been tagged.  I guess we can update this with 1.31.x / 1.30.x as backports happen.

Seems worth mentioning https://github.com/spinnaker/spinnaker/issues/6224 here too, yes?

Reply:
Looks like this will make it into 1.31.1, and once https://github.com/spinnaker/orca/pull/4478 is in, it'll be headed for 1.30.3.